staging/src/k8s.io/kubectl/pkg/cmd/apply/applyset.go

```diff
@@ -66,6 +70,10 @@ const (
 	// ApplysetPartOfLabel is the key of the label which indicates that the object is a member of an ApplySet.
 	// The value of the label MUST match the value of ApplySetParentIDLabel on the parent object.
 	ApplysetPartOfLabel = "applyset.k8s.io/part-of"
+
+	// ApplysetParentCRDLabel is the key of the label that can be set on a CRD to identify
+	// the custom resource type it defines (not the CRD itself) as an allowed parent for an ApplySet.
+	ApplysetParentCRDLabel = "applyset.k8s.io/is-parent-type"
```

Comment:
I'm not set on this specifically, but the value we had in the KEP (`applyset.k8s.io/role/parent`) is not a valid label key (only one slash allowed).

Reply:
Ah - good point!  Yes, this value seems reasonable.